packages/cli/src/eventbus/__tests__/audit-event-relay.service.test.ts

```diff
@@ -0,0 +1,83 @@
+import { mock } from 'jest-mock-extended';
+import { AuditEventRelay } from '../audit-event-relay.service';
+import type { MessageEventBus } from '../MessageEventBus/MessageEventBus';
+import type { AuditEventArgs } from '../audit.types';
+import type { EventRelay } from '../event-relay.service';
+
+describe('AuditorService', () => {
+	const eventBus = mock<MessageEventBus>();
+	const eventRelay = mock<EventRelay>();
+	const auditor = new AuditEventRelay(eventRelay, eventBus);
+
+	afterEach(() => {
+		jest.clearAllMocks();
+	});
+
+	it('should handle `user-deleted` event', () => {
+		const arg: AuditEventArgs['user-deleted'] = {
+			user: {
+				id: '123',
+				email: 'john@n8n.io',
+				firstName: 'John',
+				lastName: 'Doe',
+				role: 'some-role',
+			},
+		};
+
+		// @ts-expect-error Private method
+		auditor.userDeleted(arg);
+
+		expect(eventBus.sendAuditEvent).toHaveBeenCalledWith({
+			eventName: 'n8n.audit.user.deleted',
+			payload: {
+				userId: '123',
+				_email: 'john@n8n.io',
+				_firstName: 'John',
+				_lastName: 'Doe',
```

Comment:
Are these still needed for something before being sent? If not, I think it would make sense to move the anonymization part as part of the `AuditEventRelay`. But that can be done separately, not in this PR

Reply:
On `master` the event bus marks redactables, but it is later the log streaming destination that anonymizes redactables, based on user preferences - may or may not anonymize. But even when not anonymizing, the underscores remain anyway, which I don't like but might be a breaking change. 

I'd like to keep this PR about decoupling, but for sure let's revisit this behavior in future.